pywbem_mock/_mainprovider.py

```diff
@@ -100,25 +105,16 @@ def __init__(self, conn, datastore, default_namespace=DEFAULT_NAMESPACE):
               variables are extracted for the repository including
               the host and disable_pull_operations attributes.
 
-            datastore(:class:`~pywbem_mock.InMemoryRepository` or other subclass of `~pywbem_mock.baseRepository`):
-              class defining the data store for the CIM repository.
-
-            default_namespace(:term:`string`):
-              The default namespace defined if no namespace is specified
-              on a request.
-              TODO: Not sure this is necessary. On server side there should
-              be namespace with every request.
-        """  # noqa: E501
-
+            datastore(:class:`~pywbem_mock.InMemoryRepository` or other subclass of `~pywbem_mock.baseRepository`):  # noqa: E501
+             class defining the data store for the CIM repository.
+        """
         #   Host name required for some responses. this is the host, port
         self.host = conn.host
         self.conn = conn
 
-        self.default_namespace = default_namespace
-
         # Implementation of the CIM object repository that is the data store
         # for CIM classes, CIM instances, CIM qualifier declarations and
-        # CIM methods for the mocker.  All access to the mocker CIM data must
+        # CIM methods.  All access to the  CIM data must
         # pass through this variable to the CIM repository.
         # See :py:module:`pywbem_mock/inmemoryrepository` for a description of
         # the repository interface.
```

Comment:
We should point to `BaseRepository` for the description. At least if that sentence is needed at all, given that the datatype of the corresponding parameter already mentions that type.

Reply:
Completel removed. DONE